lib/components/parallax_component.dart

```diff
@@ -71,52 +73,52 @@ class ParallaxLayer {
     _scale = scale(parallaxImage.fill);
 
     // The image size so that it fulfills the LayerFill parameter
-    _imageSize = Offset(_image.width / _scale, _image.height / _scale);
+    _imageSize =
+        Vector2(_image.width.toDouble(), _image.height.toDouble()) / _scale;
 
     // Number of images that can fit on the canvas plus one
     // to have something to scroll to without leaving canvas empty
-    final countX = 1 + size.width / _imageSize.dx;
-    final countY = 1 + size.height / _imageSize.dy;
+    final countX = 1 + size.x / _imageSize.x;
+    final countY = 1 + size.y / _imageSize.y;
```

Comment:
if we change `count` to a Vector2 we can greatly simply the following operations. 
```
final count = (size + Vector2(1, 1))..divide(_imageSize)
```
granted `divide` a vector by a vector is a super weird an uncommon operation, so I would be ok with leaving this. the lib does support it but it's quite unclear.

could also be done manually ie

```
final count = Vector2(
    1 + size.x / _imageSize.x,
    1 + size.y / _imageSize.y,
).
```

just to reap the benefits later